tests/unit/test_plugin.py

```diff
@@ -44,7 +45,7 @@ def test_get_csp_name():
 
 
 @patch('csp_billing_adapter_google.plugin.urllib.request.urlopen')
-def test_get_account_info(mock_urlopen):
+def test_get_account_info_pass(mock_urlopen):
```

Comment:
You renamed this test to have an _pass suffix, but haven't added a version with a failure suffix... So the rename is unnecessary, until you add more tests.

Reply:
removing _pass I added because I expected to have a fail case, but get_account_info won't directly error.